auslib/web/public/api.yml

```diff
@@ -0,0 +1,539 @@
+swagger: '2.0'
+info:
```

Comment:
Is there a reason you didn't add the dockerflow, contribute.json, and robots.txt endpoints to the spec?